src/frontend/staff-portal/src/app/components/staff-workbench/contact-info/contact-info.component.html

```diff
@@ -385,16 +385,18 @@ <h4>Who are we contacting?</h4>
         <app-upload [disputeInfo]="lastUpdatedDispute"></app-upload>
         <div style="float: right; width: 100%;">
           <button class="large" mat-flat-button type="button" color="primary"
-            [disabled]="(lastUpdatedDispute.status !== DispStatus.New && lastUpdatedDispute.status !== DispStatus.Rejected) || this.form.invalid || this.form.touched"
+            [disabled]="lastUpdatedDispute.status !== DispStatus.New || this.form.invalid || this.form.touched"
```

Comment:
Should we add lastUpdatedDispute.status !== DispStatus.Validated as well since validated status is allowed?

Reply:
This condition is valid for E tickets. We don't have validate condition for E tickets